resources/views/layouts/navbar.blade.php

```diff
@@ -0,0 +1,88 @@
+<div class="container" style="background-color: #253C5C">
+
+    <a class="navbar-brand text-white" href="{{ url('/') }}">
+        
+        <img src="/images/logo.png" height="40px" class="mt-1">
```

Comment:
can you do it like this? for the img src
{{ asset('images/logo.png') }}

Reply:
I changed the img src like {{ asset('images/logo.png') }}